test/gtest/ucp/test_ucp_rma_mt.cc

```diff
@@ -78,13 +78,21 @@ UCS_TEST_P(test_ucp_rma_mt, put_get) {
     st = ucp_rkey_pack(receiver().ucph(), memh, &rkey_buffer, &rkey_buffer_size);
     ASSERT_UCS_OK(st);
 
-    ucp_rkey_h *rkey;
+    std::vector<ucp_rkey_h> rkey;
+    rkey.resize(MT_TEST_NUM_THREADS);
 
-    rkey = (ucp_rkey_h *)malloc(sizeof(ucp_rkey_h) * sender().get_num_workers());
-    for (i = 0; i < sender().get_num_workers(); i++) {
-        st = ucp_ep_rkey_unpack(sender().ep(i), rkey_buffer, &rkey[i]);
+    /* test parallel rkey unpack */
+#if _OPENMP && ENABLE_MT
```

Comment:
minor: maybe move `ENABLE_MT` before test declaration?
```
#if ENABLE_MT
UCS_TEST_P(test_ucp_rma_mt, put_get) {
```

it seems that the test doesn't make sense if MT isn't enabled

Reply:
i preferred to run this test event in ST mode to have more coverage..